Kickstarter-iOS/Features/Messages/Controller/MessagesViewController.swift

```diff
@@ -161,6 +194,51 @@ internal final class MessagesViewController: UITableViewController, MessageBanne
     self.present(vc, animated: true)
   }
 
+  private func updateTableViewBottomContentInset() {
+    if let userBlockedBannerHostingController = userBlockedBannerHostingController {
+      self.tableView.contentInset
+        .bottom = userBlockedBannerHostingController.view.bounds
+        .height
+    }
+  }
+
+  private func removeUserBlockedBanner() {
+    guard let bannerHostingController = self.userBlockedBannerHostingController,
+      let tabBarController = self.tabBarController else { return }
+
+    bannerHostingController.removeFromParent()
```

Comment:
These are missing `willMoveToParent` and `didMoveToParent` (see: [Creating a Custom Container View Controller](https://developer.apple.com/documentation/uikit/view_controllers/creating_a_custom_container_view_controller))